src/libraries/System.Globalization/tests/NlsTests/System.Globalization.Nls.Tests.csproj

```diff
@@ -222,6 +222,9 @@
     <Compile Include="$(CommonTestPath)System\RandomDataGenerator.cs"
              Link="Common\System\RandomDataGenerator.cs" />
   </ItemGroup>
+  <ItemGroup>
```

Comment:
As you suggested, I had to add an [rd.xml](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Globalization/tests/default.rd.xml#L5-L12) file to preserve `UnicodeCategory ` for the other Globalization test